app/api/auth/Farmer/register/route.ts

```
@@ -0,0 +1,88 @@
+import { connectToDatabase } from "@/lib/db";
+import Farmer from "@/models/Farmer";
+import { NextRequest, NextResponse } from "next/server";
+
+export async function POST(req: NextRequest) {
+  try {
+    const {
+      fullname: fullname,
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)